### PR TITLE
Make readme more readable

### PR DESCRIPTION
**Pull request by romanlutz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ A Python package that implements a variety of fairness-related algorithms to mit
...
-| `grid_search.binary_protected_attribute.binary_classification` | As described in Section 3.4 in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | binary | DP, EO |
-| `grid_search.binary_protected_attribute.regression` | Grid Search for regression | regression | binary | BGL |
-| `post_processing.ROCCurveBasedPostProcessing` | Post-processing algorithm based on the paper [Equality of Opportunity in Supervised Learning](https://arxiv.org/pdf/1610.02413.pdf)| binary classification | categorical | DP, EO |
+| `fairlearn.` `expgrad` | Black-box approach to fair classification described in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | categorical | DP, EO |
+| `fairlearn.` `reductions.` `GridSearch` | As described in Section 3.4 in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | binary | DP, EO |
```
Comment (riedgar-ms) [inline README.md]: No EO yet for `GridSearch` - that's what my ongoing thread with Miro about `Moment`s etc. is (in part) about.

**Comment by riedgar-ms** [comment]
Also, in this file, should we be mentioning `setup.py`? Or should we be getting rid of it? I'm afraid I don't know much about Python packaging.

File: README.md (hunk 1 above)
Comment (romanlutz) [reply README.md]: Isn't EqualizedOdds pretty simple in the sense that you just pass a bunch of constraints? I thought your discussion was about regression and incorporating BGL?

Comment (romanlutz) [reply README.md]: I've adjusted this to not mention EO. You can add it back if/when it's supported.

**Pull request merged**